src/lib.rs

```diff
@@ -308,14 +308,9 @@ where
                                 details.activity_id.clone(),
                             ));
                         } else {
-                            // TODO: Unless we return this JS seems to just give up on polling.
-                            //   -- we shouldn't need to return it?
                             warn!(task_token = ?task_token,
                                   "Unknown activity task when issuing cancel");
-                            return Ok(ActivityTask::cancel_from_ids(
-                                task_token,
-                                "unknown".to_owned(),
-                            ));
+                            continue;
```

Comment:
Should we leave a comment saying that we just ignore cancels from the heartbeat for activities that already completed?

Reply:
Yeah, I'll add that. This is actually exactly how the code was before I added this weird fake activity task, which from the old TODO you can see I didn't really want to do anyway.